clientv3/client.go

```diff
@@ -200,5 +200,6 @@ func dialEndpointList(c *Client) (*grpc.ClientConn, error) {
 }
 
 func isRPCError(err error) bool {
-	return grpc.Code(err) != codes.Unknown
+	return grpc.Code(err) != codes.Unknown &&
```

Comment:
`rpc error: code = 2 desc = "grpc: the client connection is closing"` => isRPCError() is false. I think I just screwed up the conditional when writing the fix.